InvenTree/templates/js/translated/part.js

```diff
@@ -324,20 +324,39 @@ function editCategory(pk) {
  * Delete a PartCategory via the API
  */
 function deletePartCategory(pk, options={}) {
-    var url = `/api/part/category/${pk}/`;
+    var url = `/api/part/category/${pk}/?child-categories-action=${pk}&child-parts-action=${pk}`;
```

Comment:
Why the extra URL? The fields as passed as json data in the DELETE operation, not as query parameters in the URL